docs/podtemplates.md

```diff
@@ -7,7 +7,7 @@ weight: 1400
 # Pod templates
 
 A Pod template defines a portion of a [`PodSpec`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#pod-v1-core)
-configuration that Tekton can use as "boilerplate" for a Pod that runs your `Tasks` and `Pipelines`.
+configuration that Tekton can use as "boilerplate" for a Pod that runs your `Tasks`, `Pipelines` and [Affinity Assistants](./workspaces.md#specifying-workspace-order-in-a-pipeline-and-affinity-assistants).
```

Comment:
Thanks for this.

I think it would be good to document how to configure the `Pod` template for the affinity assistant.
Looking at the code, it looks like it would be taken from the global `Pod` template config, so you could at least updates L15-19 to reflect that.

Reply:
Ended up adding a dedicated section for it: https://github.com/tektoncd/pipeline/pull/4348/files#diff-d8925b360ac9c8031196d7a047fa9b04b848577c34988d08e1396d54e48419fbR25-R38